### DISCUSSION TITLE
Invalid command when OK

### DISCUSSION THREAD

**Discussion by KoNekoD** [rabbitmq/amqp091-go#237] [Q&A]
### Describe the bug

See screenshot
![image](https://github.com/rabbitmq/amqp091-go/assets/108808201/19e3b63d-adc5-4995-9c3b-23c82b718414)

### Reproduction steps

1. Try declare multiple elements in one connection

### Expected behavior

Not unexcepted

**Comment by KoNekoD** [comment]
![image](https://github.com/rabbitmq/amqp091-go/assets/108808201/01da6c40-54f2-46e7-b2c2-957d0d72535e)

**Comment by lukebakken** [comment]
I can't tell what is going on here.

Instead of screenshots, please provide a code sample that I can clone, compile, and run to reproduce your issue. The code sample must be *COMPLETE* - I do not have time to try to get something to run when providing _free_ support. Thank you.
